build.gradle

```diff
@@ -131,3 +131,10 @@ shadowJar {
 }
 
 assemble.dependsOn(shadowJar)
+
+// Invoke arbitrary classes interactively. Example:
+// ./gradlew runApp -P mainClass=io.crate.flink.demo.SimpleTableApiJob
+task runApp(type:JavaExec) {
+    classpath = sourceSets.main.runtimeClasspath
+    main = project.hasProperty("mainClass") ? project.getProperty("mainClass") : "package.MyDefaultMain"
```

Comment:
I'd rather use a different name, like `demoJob` without a default fallback,
in order not to mix up with gradle's `mainClassName`, wdyt?

Reply:
Addressed with 42901f80ad.

Reply:
I meant to change `mainClass` name not the `runApp`.
Imho it will look better as `demoJob` and `runJob` respectively.

Reply:
Like this?

```groovy
// Invoke arbitrary classes interactively. Example:
// ./gradlew runJob -P demoJob=io.crate.flink.demo.SimpleTableApiJob
task runJob(type:JavaExec) {
    classpath = sourceSets.main.runtimeClasspath
    main = project.getProperty("demoJob")
}
```

Reply:
See 75f199acca.

Reply:
yes, thx!